### PR TITLE
Initial write up of details to get from people

### LINKED ISSUE #35: Create project workshop requirements form
### Detail

Creating a form to help us determine what the nature of a particular project workshop is, including:
- A short summary of the session
- Any resources people should familiarise themselves before hand
- What kind of people should join the discussion

### Actions

- [ ] Create the template
- [ ] Create comms to accompany the template
- [ ] Share with December project workshop people

### Who can help

_No response_

### PR DESCRIPTION
**Pull request by harisood**
Fixes #35

### CODE REVIEW

**Comment by harisood** [comment]
Whoops I think I created this from the wrong branch, let's finalise and merge #26 and come back to this

File: events/templates/prompts_for_sessions.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Prompts for sessions in quarterly meetings
+
+At the quarterly meetings we ask registrants whether they would like to host a `Project workshop` or `Breakout discussion` on the day.
+
+Those who say yes are sent the following prompts to gather info for their sessions. 
+This is sent to them, and organised, via email.
+
+## Project workshops
+
```
Comment (Davsarper) [inline events/templates/prompts_for_sessions.md:9]:
```suggestion

Project workshops are focused discussions on a specific project, issue or challenge.
They are intended to serve as a collaborative work sessions where the community helps the presenter to move forward in their project. 

```

Comment (Davsarper) [reply events/templates/prompts_for_sessions.md:9]: I think that while we figure out an intuitive terminology we should add details about how we envision each type of breakout

File: events/templates/prompts_for_sessions.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+### Summary
+*Please provide a short paragraph summarising your proposed session, including background and what you hope to cover. We'll share this on our website!*
```
Comment (Davsarper) [inline events/templates/prompts_for_sessions.md:11]:
```suggestion
*Please provide a short paragraph summarising your proposed session, including background and what you hope to cover. 
We'll share this on our website!*
```

File: events/templates/prompts_for_sessions.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+### Required preparation
+*Please share anything people should read, know or prepare in advance for the session, or anything they should bring with them*
+
+### Target audience
+*If your session is aimed at a specific audience, please share details here*
+
+## Breakout discussions
+
```
Comment (Davsarper) [inline events/templates/prompts_for_sessions.md:20]:
```suggestion

Breakout discussions are open talks around a topic.
They are open ended, and while we hope actions and collaboration arise from them, there is no specific output expected by the end of the session.

```

**Review by Davsarper — CHANGES_REQUESTED** [review]

**Comment by harisood** [comment]
@Davsarper nice edits! Please review again and if happy approve :)

**Review by Davsarper — APPROVED** [review]

**Pull request merged**